src/parser/parse_tree_cl.c

```diff
@@ -18384,3 +18400,102 @@ pt_print_dblink_table (PARSER_CONTEXT * parser, PT_NODE * p)
 
   return q;
 }
+
+
+
+static PT_NODE *
+pt_apply_create_server (PARSER_CONTEXT * parser, PT_NODE * p, void *arg)
+{
+  PT_APPLY_WALK (parser, p->info.create_server.server_name, arg);
+  /*
+     PT_NODE *server_name;
+     PT_NODE *host;
+     PT_NODE *port;  
+     PT_NODE *dbname;  
+     PT_NODE *user;
+     PT_NODE *pwd;
+     PT_NODE *prop;
+     PT_NODE *comment;
+   */
```

Comment:
I don't understand what's the meaning of this comment.

Reply:
It is a list of variables belonging to create_server, and it was written for reference.
I will delete it.